src/evaluate/info.py

```diff
@@ -31,6 +31,29 @@
 logger = get_logger(__name__)
 
 
+@dataclass
+class Config:
+    """Base class to store the configuration used for the evaluation module."""
+
+    name = "default"
```

Comment:
Maybe add a comment (or docstring) that explains what `name` and `allowed_names` are and how they're related?